classes/template/PKPTemplateManager.inc.php

```diff
@@ -942,6 +1007,39 @@ function smartyLoadStylesheet($params, $smarty) {
 	}
 
 	/**
+	 * Smarty usage: {load_script context="backend" scripts=$scripts}
+	 *
+	 * Custom Smarty function for printing scripts attached to a context.
+	 * @param $params array associative array
+	 * @param $smarty Smarty
+	 * @return string of HTML/Javascript
+	 */
+	function smartyLoadScript($params, $smarty) {
+
+		if (empty($params['scripts'])) {
+			return;
```

Comment:
Actually it's not. A template will call `{load_script}` but a particular theme might not choose to register any.